components/dropdown/DropdownActions.js

```diff
@@ -1,44 +1,60 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Group, ButtonGroup } from 'react-components';
+import { Button, Group, ButtonGroup, Icon, Tooltip } from 'react-components';
 import { c } from 'ttag';
 
 import Dropdown from './Dropdown';
 import DropdownMenu from './DropdownMenu';
 import DropdownButton from './DropdownButton';
 
-const DropdownActions = ({ list, className }) => {
+const wrapTooltip = (text, tooltip) => {
+    if (!tooltip) {
+        return text;
+    }
+    return (
+        <>
+            {text}{' '}
```

Comment:
Try to not use `{' '}`, prefer:
```suggestion
            <span className="mr0-5">{text}</span>
```